src/env.ts

```diff
@@ -23,4 +23,7 @@ export default envalid.cleanEnv(process.env, {
   USER_URI: envalid.str({ devDefault: '//Alice' }),
   IPFS_HOST: envalid.host({ devDefault: 'localhost' }),
   IPFS_PORT: envalid.port({ default: 5001 }),
+  SUBSTRATE_STATUS_POLL_PERIOD_MS: envalid.num({ default: 10 * 1000 }),
```

Comment:
just wonder maybe we could generalise this one and call something like `WATCHER_POLL_PERIOD_MS` - first thought pretty sure it could have some more relevant name.

Mainly because we will be polling other services pretty much save interval. If we won't then we would get some discrepancies or would need to define multiple env vars